code/game/machinery/newscaster.dm

```diff
@@ -865,14 +865,17 @@ GLOBAL_LIST_EMPTY(allCasters)
 	NEWSPAPER.creationTime = GLOB.news_network.lastAction
 	paper_remaining--
 
+
+/obj/machinery/newscaster/proc/remove_alert()
+	alert = FALSE
+	update_icon()
+
 /obj/machinery/newscaster/proc/newsAlert(channel)
 	if(channel)
 		say("Breaking news from [channel]!")
-		alert ++
+		alert = TRUE
 		update_icon()
-		spawn(alert_delay)
-			alert --
```

Comment:
wait okay does this remove all alerts if one alert expires right after another is created?

Reply:
No, TIMER_OVERRIDE causes new timer to be created over the old one effectively resetting it.